tests/Unit/DataStructures/Test_Point.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/Point.hpp"
+
+TEST_CASE("Unit.DataStructures.Point", "[DataStructures][Unit]") {
+  const size_t Dim = 3;
+
+  Point<Dim, Frame::Grid> p1{{{2.3, -1.4, 0.2}}};
+  Point<Dim, Frame::Grid> p2{{{-12.4, 4.5, 2.6}}};
+  const auto expected_plus = Point<Dim, Frame::Grid>{{{-10.1, 3.1, 2.8}}};
+  const auto expected_minus = Point<Dim, Frame::Grid>{{{14.7, -5.9, -2.4}}};
+
+  const auto plus = p1 + p2;
+  const auto minus = p1 - p2;
+
+  for (size_t i = 0; i < Dim; ++i) {
+    CHECK(plus.get(i) == Approx(expected_plus.get(i)));
+    CHECK(minus.get(i) == Approx(expected_minus.get(i)));
```

Comment:
Probably want to check these to a tighter tolerance than 1e-5 or so? I'd say this can just stay as is until #84 is resolved and we have a uniform way of handling tolerances.